src/uu/seq/src/extendedbigdecimal.rs

```diff
@@ -280,11 +273,6 @@ mod tests {
         assert_eq!(format!("{}", ExtendedBigDecimal::Infinity), "inf");
         assert_eq!(format!("{}", ExtendedBigDecimal::MinusInfinity), "-inf");
         assert_eq!(format!("{}", ExtendedBigDecimal::Nan), "nan");
-        // FIXME In Rust version 1.53.0 and later, the display of floats
-        // was updated to allow displaying negative zero. Until then, we
-        // just display `MinusZero` as "0.0".
-        //
-        //     assert_eq!(format!("{}", ExtendedBigDecimal::MinusZero), "-0.0");
-        //
+        assert_eq!(format!("{}", ExtendedBigDecimal::MinusZero), "-0");
```

Comment:
This assert works but if we change it to this:
`assert_eq!(format!("{:?}", ExtendedBigDecimal::MinusZero), "-0.0");`
it will explain better what we are looking for. Not sure how to test these assets though